src/parser.rs

```diff
@@ -62,6 +76,14 @@ pub struct BasicParseError<'i> {
     pub location: SourceLocation,
 }
 
+impl<'i> fmt::Display for BasicParseError<'i> {
+    fn fmt(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+        write!(formatter, "{}", self.kind.description())
```

Comment:
This won't print enough information, i.e. we end up not printing the payloads of `UnexpectedToken(_)` and `AtRuleInvalid(_)`.